src/pkg/Microsoft.DotNet.ILCompiler/Microsoft.DotNet.ILCompiler.pkgproj

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <PropertyGroup>
+    <PackageTargetRuntime/>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <SkipPackageFileCheck>true</SkipPackageFileCheck>
+    <BaseLinePackageDependencies>false</BaseLinePackageDependencies>
+    <PackagePlatforms>x64;</PackagePlatforms>
+    <PreventImplementationReference>true</PreventImplementationReference>
+    <SkipValidatePackage>true</SkipValidatePackage>
+
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\..\ILCompiler\src\ILCompiler.csproj"/>
+    <ProjectReference Include="..\..\ILCompiler.Compiler\src\ILCompiler.Compiler.csproj"/>
+    <ProjectReference Include="..\..\ILCompiler.DependencyAnalysisFramework\src\ILCompiler.DependencyAnalysisFramework.csproj"/>
+    <ProjectReference Include="..\..\ILCompiler.MetadataTransform\src\ILCompiler.MetadataTransform.csproj"/>
+    <ProjectReference Include="..\..\ILCompiler.MetadataWriter\src\ILCompiler.MetadataWriter.csproj"/>
+    <ProjectReference Include="..\..\ILCompiler.TypeSystem\src\ILCompiler.TypeSystem.csproj"/>
+
+    <Dependency Include="Microsoft.DotNet.ObjectWriter">
+      <Version>1.0.13-prerelease-00001</Version>
+    </Dependency>
+    <Dependency Include="Microsoft.NetCore.Jit">
+      <Version>1.1.0-preview1-24512-00</Version>
```

Comment:
Updated (to 1.2.0-beta-24815-03 since it looks like that's the new [latest](https://github.com/dotnet/corert/blob/master/src/packaging/packages.targets#L25)).